addon/components/frost-modal-input.js

```diff
@@ -11,6 +11,8 @@ export default Component.extend({
   $containerEl: null,
   $headerEl: null,
   $footerEl: null,
+  renderers: {},
+  validators: [],
```

Comment:
I thought it was generally a bad idea to initialize anything but simple types like this. I realize we're not doing anything like `renderers.push()`, and it's always re-assigned, so it probably won't bite us, but I thought it was worth mentioning.